### PR TITLE
fix problems with broken wheels.galaxyproject.org

### PR DESCRIPTION
**Pull request by drosofff**
This PR is a turn around of the  issue #235

### CODE REVIEW

File: tasks/galaxy_root.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@
 - name: "Install watchdog for galaxy"
   pip:
     name: "watchdog"
+    version: 0.9.0
```
Comment (mvdbeek) [inline tasks/galaxy_root.yml]: I think you need to fix the pip extra args as in https://github.com/galaxyproject/ansible-galaxy/blob/master/tasks/dependencies.yml#L14 ... but I think with a recent ansible-galaxy you don't need these tasks anymore. I would skip this entirely, but at the minimum you can't pin the watchdog version here.

File: tasks/slurm.yml [hunk 2]
```diff
@@ -63,7 +63,7 @@
 - name: Fetch DRMAA wheel for Galaxy
   pip:
     name: "drmaa"
-    extra_args: "--index-url https://wheels.galaxyproject.org/simple/"
+    version: 0.7.9
```
Comment (mvdbeek) [inline tasks/slurm.yml]: Same here, we shouldn't pin any versions here, fixing the extra_args should be sufficient to get this to run.

**Comment by bgruening** [comment]
We need to merge in the 19.05 branch. This one already fixes some things of 19.09 including wheels.

**Comment by drosofff** [comment]
@bgruening Yes, it isn't even necessary to ever merge this branch if the issue is better addressed in 19.05. It was a kind of emergency patch for a galaxy training.

**Pull request closed without merging**